lighthouse-core/test/runner-test.js

```diff
@@ -609,6 +609,27 @@ describe('Runner', () => {
     assert.ok(lhr.runtimeError.message.includes(NO_FCP.message));
   });
 
+  it('internationalized thrown errors from driver', async () => {
+    const erroringDriver = {...driverMock, connect:
```

Comment:
the formatting on here is a bit confusing. `async` could also help. What about something like
```js
const erroringDriver = {
  ...driverMock,
  async connect() {
    const err = new LHError({
      ...LHError.errors.PROTOCOL_TIMEOUT,
      protocolMethod: 'Method.Failure',
    });
    throw err;
  },
};
```